### PR TITLE
WIP: Add test to check that spelling error markers are shown

### PR DESCRIPTION
**Pull request by mrego**

### CODE REVIEW

File: html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<meta charset="utf-8">
+<title>Reference file for spellcheck tests</title>
+<style>
+  div {
+    padding: 10px;
+    background: silver;
+  }
+</style>
+
+<p>The test passes if you see a spelling error marker in the word "quikc".</p>
```
Comment (foolip) [inline html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html:12]: Can you add a comment that this ref is used as a mismatch, to help explain why the text here seems to contradict the following `spellcheck=false`?

File: html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<div id="test" contenteditable=true spellcheck=false>The quikc brown fox.</div>
+
+<script>
+  var test = document.getElementById("test");
+  test.addEventListener("focus", () => {
+    document.documentElement.classList.remove("reftest-wait");
```
Comment (foolip) [inline html/editing/editing-0/spelling-and-grammar-checking/references/spelling-markers-011-ref.html:18]: The ref here can be simplified by not bothering with the focus or "reftest-wait" at all. Or is there a cursor that needs to end up in the right place? If so, can the blinking cursor not cause the test to fail, and can it be hidden?

**Comment by mrego** [comment]
Sorry this was mostly an experiment, I'm trying to understand why I see the spelling markers in Chrome in some cases (https://github.com/web-platform-tests/wpt/pull/30817) but not in other like in this one.

content_shell doesn't show the error markers it seems, and Chrome does it lazily but still adding a timeout didn't help either.

So this is more an experimental PR or WIP, I'm trying to find the best way to test this, and if possible do it in WPT.

**Comment by foolip** [comment]
I see! A tip for experimenting is creating draft PRs. Then no reviewers will be added and you can try any number of things in CI until you're happy with it.

**Comment by mrego** [comment]
Yeah, sorry about that, not sure if I can make it a draft now. I'll close it for now and open it again as a draft if needed. Thanks for the tip.

**Pull request closed without merging**